inlineplz/linters/jscs.py

```diff
@@ -0,0 +1,58 @@
+# -*- coding: utf-8 -*-
+from __future__ import absolute_import
+from __future__ import unicode_literals
+
+import dirtyjson as json
+import os.path
```

Comment:
[inline-plz]: `prospector: pylint: standard import "import os.path" should be placed before "import dirtyjson as json" (wrong-import-order)`